nodejsalert-ui/config.js

```diff
@@ -0,0 +1,36 @@
+var config = {
+development: {
+
+
+    database: {
```

Comment:
Please make sure these are all configurable and I think these dbuser / password shouldn't saved here but should be parameter placeholders

Reply:
the parameters are configurable , and config.js file is main configuration file for node.js project. However I removed my local development sql credential from file.